helpers/setup.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import os
+
+kubectl_cmd='microk8s.kubectl'
```

Comment:
hack with classic for now. need to make this work seamlessly with kubectl.

Also, error messages and detailed help still TBD.